omod/src/main/java/org/openmrs/module/ssemrws/constants/SharedConstants.java

```diff
@@ -227,6 +227,13 @@ public static long calculateAge(Date birthdate) {
 		return (currentDate.getTime() - birthdate.getTime()) / (1000L * 60 * 60 * 24 * 365);
 	}
 	
+	public static int getPatientAge(Patient patient) {
+		Date birthdate = patient.getBirthdate();
```

Comment:
Why the decision to work with `Date` as opposed to `LocalDate`?

Reply:
@nelsonkimaiga I have resolved this.